test/jdk/javax/swing/JInternalFrame/6725409/bug6725409.java

```diff
@@ -21,63 +21,59 @@
  * questions.
  */
 
-/* @test
+/*
+ * @test
  * @key headful
  * @bug 6725409
  * @requires (os.family == "windows")
  * @summary Checks that JInternalFrame's system menu
  *          can be localized during run-time
- * @author Mikhail Lapshin
  * @library /lib/client/
  * @modules java.desktop/com.sun.java.swing.plaf.windows
- * @build ExtendedRobot
  * @run main bug6725409
  */
 
-import javax.swing.*;
-import java.awt.*;
+import java.awt.Robot;
+import javax.swing.JDesktopPane;
+import javax.swing.JFrame;
+import javax.swing.JInternalFrame;
+import javax.swing.JMenuItem;
+import javax.swing.JPopupMenu;
+import javax.swing.SwingUtilities;
+import javax.swing.UIManager;
+import javax.swing.UnsupportedLookAndFeelException;
 
 public class bug6725409 {
     private JFrame frame;
     private JInternalFrame iFrame;
-    private TestTitlePane testTitlePane;
+    private static TestTitlePane testTitlePane;
     private boolean passed;
-    private static ExtendedRobot robot = createRobot();
+    private static final Robot robot = createRobot();
 
     public static void main(String[] args) throws Exception {
         try {
             UIManager.setLookAndFeel(
                     new com.sun.java.swing.plaf.windows.WindowsClassicLookAndFeel());
-        } catch(UnsupportedLookAndFeelException e) {
+        } catch (UnsupportedLookAndFeelException e) {
             System.out.println("The test is for Windows LaF only");
             return;
         }
 
         final bug6725409 bug6725409 = new bug6725409();
         try {
-            SwingUtilities.invokeAndWait(new Runnable() {
-                public void run() {
-                    bug6725409.setupUIStep1();
-                }
-            });
+            SwingUtilities.invokeAndWait(() -> bug6725409.setupUIStep1());
```

Comment:
```suggestion
            SwingUtilities.invokeAndWait(bug6725409::setupUIStep1);
```
Since you're still updating these lines, I'm for using method references (in all the cases).